docs/supportedsites.md

```diff
@@ -619,6 +620,7 @@
  - **VideoTt**: video.tt - Your True Tube
  - **videoweed**: VideoWeed
  - **Vidme**
+ - **vidto**: VidTo.me
```

Comment:
This file is updated automatically in each stable release. Please do not change its content in normal commits.